lasagne/regularization.py

```diff
@@ -1,3 +1,53 @@
+"""
+Functions to apply regularization to the weights in a network.
+
+We provide functions to calculate the L1 and L2 penalty. Penalty functions
+take a tensor as input and calculate the penalty contribution from that tensor:
+
+.. autosummary::
+    :nosignatures:
```

Comment:
Hmm, not sure about changing the signature, we also have `x` in the `nonlinearities` module: http://lasagne.readthedocs.org/en/latest/modules/nonlinearities.html
And numpy also just uses `x`, e.g., in `np.square`: http://docs.scipy.org/doc/numpy/reference/generated/numpy.square.html
I think it's clear enough the way it is (but I still like `l1(tensor)`). @ebenolson, since you wrote this, do you have any opinion?

Otherwise let's just leave it as it is! (Without any signatures in the table, and with `x` instead of `tensor`.) The `apply_penaly` signature really seems too long for the summary table.